skills/plane-backlog/scripts/plane_create_issue.py

```
@@ -0,0 +1,500 @@
+        res = subprocess.run(cmd, capture_output=True, text=True, check=True)
+        for line in res.stdout.splitlines():
+            if line.startswith("RESULT_JSON:"):
+                return json.loads(line[len("RESULT_JSON:"):])
+        return {"success": False, "reason": f"No RESULT_JSON line output. Stdout: {res.stdout}, Stderr: {res.stderr}"}
+    except subprocess.CalledProcessError as e:
+        return {"success": False, "reason": f"K3s execution failed: {e.stderr or e.stdout or str(e)}"}
+    except Exception as e:
+        return {"success": False, "reason": f"K3s execution failed: {str(e)}"}
+
+
+def create_plane_issue(title: str, description: str = "", project_id: str = None, is_intake: bool = True, cwd: str = None) -> dict:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)